ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-BL1A.cfg

```diff
@@ -6,6 +6,8 @@ daemon TerminAttr
 !
 vlan internal order ascending range 1006 1199
 !
+ip igmp snooping
```

Comment:
We shouldn't be configuring ip igmp snooping even if this is the default.

Reply:
I missed it, while refactoring. Will add if check.

Reply:
Actually, I added the logic to print only if the user defines it true. That's not generated by default.

Reply:
If this is causing breaking changes, I will remove that command from the template. So that this config won't be generated even if someone defines it true.

Reply:
Removed it!